tools/upgrade_versions.py

```diff
@@ -254,6 +255,8 @@ def run(CONF):
         pkg_version.update(updater_name, updater_email)
         pkg.unlock()
 
+    update_versions_in_readme(versions_repo, distro, packages_to_update)
```

Comment:
This will create a table with only the packages being updated. We'll want a complete table instead, so you could either pass `config.discover_packages()` or discover the packages inside the function and disallow generating incomplete tables.